tests/rewards/test_reward_nets.py

```diff
@@ -30,6 +36,27 @@ def test_init_no_crash(env_name, reward_net_cls, reward_net_kwargs):
         reward_net_cls(env.observation_space, env.action_space, **reward_net_kwargs)
 
 
+@pytest.mark.parametrize("env_name", ENVS)
+@pytest.mark.parametrize(
+    "reward_net_kwargs",
+    [
+        {"rew_normalize_class": networks.RunningNorm},
+        {
+            "normalize_input_layer": networks.RunningNorm,
+            "rew_normalize_class": networks.RunningNorm,
+        },
+    ],
+)
+def test_init_normalized_reward_net_no_crash(env_name, reward_net_kwargs):
+    env = gym.make(env_name)
+    for i in range(3):
```

Comment:
I'm skeptical doing this three times really tests anything more (although I see we do this in an existing test too)